index.html

```diff
@@ -214,29 +214,29 @@ <h2>On behalf of organisations</h2>
 					<li>Les utilisateurs et utilisatrices naviguant depuis Google vers un contenu recommandé par Google restent, sans le savoir, dans l’enceinte de l’écosystème Google.</li>
 				</ol>
 
-				<p>Si l’objectif de Google avec AMP est en réalité d’améliorer l’expérience utilisateur sur le web, alors nous suggérons de simples changements qui serviraient cette cause tout en permettant au web de rester dynamique, compétitif et tourné vers les consommateurs:</p>
+				<p>Si l’objectif de Google avec AMP est réellement d’améliorer l’expérience utilisateur sur le web, alors nous suggérons de simples changements qui serviraient cette cause tout en permettant au web de rester dynamique, compétitif et tourné vers les consommateurs:</p>
 
 				<ol>
 					<li>Au lieu de réserver un placement préférentiel pour les seuls contenus AMP dans les résultats de recherche, il suffirait de procurer des privilèges similaires à toute page qui satisferait à des critères de performance neutres et objectifs, comme, par exemple, le Speed Index. Les éditeurs pourront ainsi utiliser la solution technique de leur choix.</li>
-					<li>Ne pas intégrer de contenus tiers externes dans une page Google, à moins qu’il ne soit clair pour l’utilisateur ou l’utilisatrice d’avoir affaire à un produit Google. Il est parfaitement acceptable que Google lance un “lecteur de nouvelles”, mais il n’est ni acceptable de servir une page n’affichant que du contenu tiers depuis une URL de Google, ni d’obliger le tiers à utiliser l’hébergement de Google pour apparaître plus en évidence dans les résultats de recherche.</li>
+					<li>Ne pas intégrer de contenus tiers externes dans une page Google, à moins qu’il ne soit clair pour l’utilisateur ou l’utilisatrice d’avoir affaire à un produit Google. Il est parfaitement acceptable que Google lance un “lecteur de nouvelles”, mais il n’est ni acceptable de servir une page n’affichant que du contenu tiers depuis une URL de Google, ni d’obliger le tiers à utiliser l’hébergement de Google pour apparaître dans les résultats de recherche.</li>
 				</ol>
 
-				<p>Nous ne cherchons pas à stopper le développement de AMP par Google, et ces changements ne le requièrent pas. Aussi, nous approuvons la promotion par les moteurs de recherche des pages performantes. AMP peut demeurer l’une des technologies qui donnent aux éditeurs le choix entre une variété d’options de grande qualité afin de servir des pages Web rapidement et de rendre leur utilisation plus satisfaisante.</p>
+				<p>Nous ne cherchons pas à stopper le développement de AMP par Google, et ces changements ne le requièrent pas. Aussi, nous approuvons la promotion par les moteurs de recherche des pages performantes. AMP peut demeurer l’une des technologies qui donnent aux éditeurs le choix entre une variété d’options de grande qualité afin de servir des pages Web rapidement et de rendre leur utilisateurs plus satisfaits.</p>
```

Comment:
👎

Less inclusive and introduces a grammar error.

Reply:
I agree on inclusiveness. No grammar error IMO, and semantics is closer to `making users happy`. I reverted (and rebased) anyway. :)

Reply:
There was a _faute d'accord_: "leur utilisateurs plus satisfaits" -> leur**s**